checks/cloud/openstack/networking/add_description_to_security_group.rego

```diff
@@ -26,8 +26,14 @@ package builtin.openstack.networking.openstack0005
 
 import rego.v1
 
+import data.lib.cloud.value
+
 deny contains res if {
 	some sg in input.openstack.networking.securitygroups
-	sg.description.value == ""
-	res := result.new("Security group rule allows egress to multiple public addresses.", sg.description)
+	without_description(sg)
+	res := result.new("Network security group does not have a description.", sg.description)
```

Comment:
This change should have failed a test right? (since the resulting message has changed).

Reply:
In some tests, we only check the number of results, not the messages.

Reply:
OK - I think we should improve that, at least for those checks where there isn't any more than a single message that can be in the output. We can do it in a separate PR if you like.